frontend/auth/signup.html

```diff
@@ -1,8 +1,8 @@
 <div layout="row" layout-align="center center">
   <md-button class="md-fab md-primary" aria-label="Retornar" 
-    style="margin-right: -27px; width: 65px; height: 65px;" md-colors="{background: 'light-green-600'}"
+    style="margin-right: -27px;" md-colors="{background: 'light-green-600'}"
```

Comment:
move style from template to css file

Reply:
The same in the other cases

Reply:
This page wasn't made the refactoring, I fixed only the button.